tests/drivers/console/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Please switch to `find_package(Zephyr)`
```suggestion
find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
```

---

Done